File: noi.openjudge.cn/ch0105/01.cpp
// http://noi.openjudge.cn/ch0105/solution/39324248/

#include<iostream>
using namespace std;
int main() {
    int n,a;
    double b,sum = 0;
    cin >> n;
    for (int i=0;i<n;i++) {
        cin >> a;
        b = a;
        sum += b;
    }
    printf("%0.2f\n",sum/n);
}

File: noi.openjudge.cn/ch0110/01.py
# http://noi.openjudge.cn/ch0110/solution/39973031/

n, m = map(int,input().split())
a = []
for i in range(n):
    x, y = input().split()
    z = float(y)
    a.append((z, y, x))
a.sort(reverse=True)
x = a[m-1][2]
y = a[m-1][1]
print(x, y)
